### ISSUE TITLE
As a user, I want full historical data imported for metrics

### ISSUE THREAD

**Issue by bobheadxi** [pullreminders/backlog#94]
First of all, I love the new dashboard stuff!

However, would it be possible to view longer time periods? The limit seems to be 24 weeks - at [UBC Launch Pad](https://github.com/ubclaunchpad), as a school club, this time frame doesnt make too much sense as our activity drops down in the summer, and the graphs dont tell us if we are making significant year-on-year improvements.

If a complete history is not possible, a 53-week (1 year) time frame would be useful too for insights into year-on-year improvements!

**Comment by abinoda** [comment]
@bobheadxi Thanks for filing this Robert! The issue with longer data history is really just a matter of cost for us right now. Both in terms of storage space and the additional compute resources required to go *all the way back*. We're actively thinking about this problem though, including potentially making it an additional paid add-on or something. Where its desperately wanted or needed, we can run a special import for accounts to go back further in time, though.

**Comment by bobheadxi** [comment]
> Both in terms of storage space and the additional compute resources required to go all the way back

This is pretty understandable, though even just one year (52 weeks) would be nice as well! Alternatively, "condensed" versions of older historical data might be less taxing while still allowing *some* data to be available

**Comment by abinoda** [comment]
@bobheadxi Yep – we're working on some behind-the-scenes database cleanup stuff so we'll reassess this once we have a better look at actual storage costs.
